catroid/src/main/java/org/catrobat/catroid/content/Scene.java

```diff
@@ -75,56 +69,91 @@
 	@XStreamAlias("data")
 	private DataContainer dataContainer = null;
 	@XStreamAlias("originalWidth")
-	private int originalWidth = 0;
+	public int originalWidth = 0;
 	@XStreamAlias("originalHeight")
-	private int originalHeight = 0;
+	public int originalHeight = 0;
```

Comment:
Are we serializing unused fields here or do (will) they have any purpose? (I'm not up-to-date with the different screen size handling…)

Why the change though?

Reply:
I have absolutely no idea what it does, but it is part of the code.xml and because I did not want to change the xml parsing, I left the variable and just removed the unused getters, which then caused a checkstyle or lint (I don't remember which one it was) error because you're not supposed to have unused private variables (obviously).

Reply:
Ah, now I get it, thanks!